cmd/up/up.go

```diff
@@ -372,21 +381,20 @@ func Up() *cobra.Command {
     https://www.okteto.com/docs/reference/manifest-migration/`))
 			}
 
-			err = up.start()
-
-			if err != nil {
+			if err = up.start(); err != nil {
 				switch err.(type) {
 				default:
-					err = fmt.Errorf("%w\n    Find additional logs at: %s/okteto.log", err, config.GetAppHome(dev.Namespace, dev.Name))
+					return fmt.Errorf("%w\n    Find additional logs at: %s/okteto.log", err, config.GetAppHome(dev.Namespace, dev.Name))
 				case oktetoErrors.CommandError:
 					oktetoLog.Infof("CommandError: %v", err)
+					return err
 				case oktetoErrors.UserError:
 					return err
 				}
-
 			}
 
-			return err
+			up.analyticsMeta.CommandSuccess()
```

Comment:
if `up` fails that metric is sent? or do we assume that if the `success` metric does not exist it has failed?

Reply:
i've wrapped the err check into line 384, `CommandSuccess` sets the property `success` to true, if some error occur this is not set so the cmd is reportted as success false.